### DISCUSSION TITLE
Missing issue references in GitHub release notes

### DISCUSSION THREAD

**Discussion by ahus1** [community/community#32650] [Other Feature Feedback, Questions, & Ideas]
Today I found that the most recent releases don't render links to the issues in the markdown description. 

Content - braces include the issue number, and I expect them to be rendered as links as in previous releases:
```
- Improve highlighting of the pass macro
- Show fewer errors in blocks that are excluded via ifdef/ifndef (#1150)
- Fix the option to hide validation errors in source blocks in 2022.2 (#1151)
- Update Antora CSS theme and fonts to the latest from Antora UI
```

Preview - braces are empty: 

![image](https://user-images.githubusercontent.com/3957921/189524806-d3639761-9479-4ffe-ab87-c56356d7a40d.png)

Is this a bug?

Example: https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases/tag/0.37.46

**Comment by ychin** [comment]
I'm seeing that as well and it seems new. I was editing some of my older release notes to give them better formatting at https://github.com/macvim-dev/macvim/releases, and the edited ones all lost their issue tags in the Markdown rendering. Meanwhile older releases that I didn't touch still have the issue tags. Seems like they may have introduced a bug to the renderer?

**Reply by ychin** [reply]
Seems like this is fixed. I didn't get a response to my bug report yet but looking at my releases I now see the issue number tag again.

**Comment by airtower-luna** [comment]
That looks bad! :sweat_smile: 

Did you [file a bug report](https://support.github.com/contact/bug-report) yet?

**Reply by ychin** [reply]
I just did!
